### PR TITLE
Add image to about me section

### LINKED ISSUE #5: Add Image to 'About Me' Section with Best Practices
#### Description
Include an image in the 'About Me' section, ensuring it adheres to best practices for accessibility and optimization.

#### Tasks
- [x] Select an appropriate image for the 'About Me' section.
- [x] Add the image to the HTML structure with descriptive alt text.
- [x] Optimize the image for web using appropriate file formats and sizes. [use your own picture]

Screenshot 2023-12-06 at 4 48 28 PM

### PR DESCRIPTION
**Pull request by MasonPartridge**
## Changes

Add image to about me:
about-me-image-pull

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -27,8 +27,24 @@ a {
   display: flex;
 }
 
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.max-w-532px {
```
Comment (alexrdrgz) [inline css/style.css:34]: Really like this technique and naming convention

File: css/style.css [hunk 2]
```diff
@@ -27,8 +27,24 @@ a {
...
+  max-width: 532px;
+}
+
+.shrink-0 {
+  flex-shrink: 0;
```
Comment (alexrdrgz) [inline css/style.css:39]: This is pretty cool, never used this before

File: css/style.css [hunk 3]
```diff
@@ -9,7 +9,7 @@ a {
   margin-right: 45px;
   margin-left: 45px;
   font-size: 16px;
-  font-family: 'Inter', sans-serif;
+  font-family: "Inter", sans-serif;
```
Comment (alexrdrgz) [inline css/style.css:12]: Maybe add this to the body? I think i used the same font on everything right?

**Review by alexrdrgz — COMMENTED** [review]
Really good PR, I like the way you created reusable classes In anticipation of the rest of the project.

To get the fulll benefits of that, just make sure to inform your team and let them know of why you created those classes and when you should use them. Perhaps comment it out if they will be reused often.

**Review by BSchoolland — APPROVED** [review]
The styling is off a bit off, but this can be fixed in a later issue.

**Pull request merged**